.travis.yml

```diff
@@ -12,9 +12,22 @@ before_script:
   - unset PYTHON_CFLAGS # HACK
   - ./configure CFLAGS=-Werror --with-ivykis=internal --prefix=$HOME/install/syslog-ng --enable-pacct --enable-manpages --with-docbook=/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/docbook.xsl 
   - if [ "$CC" = "clang" ]; then export COPYRIGHTVERBOSITY=1; make check-copyright; fi
-  - make -j || make V=1 --keep-going # to make error messages more readable on error
+  - . tests/build-log-cflags-propagation.sh;
+    exec_prop_check "make -j V=1";
+    S=$?;
+    if [ "$S" = "42" -o "$S" = "0" ]; then
+      return $S;
```

Comment:
We retunn 42 if propagation check failed, but the build otherwise succeeded. 0 means both succeeded. Other values signal a build failure. We could use another extremal value if you prefer.